src/Surfnet/StepupRa/RaBundle/Form/Type/VerifyIdentityType.php

```diff
@@ -21,18 +21,22 @@
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Length;
 
 class VerifyIdentityType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder->add('documentNumber', 'text', [
             'label' => 'ra.form.verify_identity.document_number.label',
+            'constraints' => new Length(6),
             'horizontal_label_class' => 'col-sm-6 left-aligned',
             'horizontal_input_wrapper_class' => 'col-sm-6',
             'attr' => [
                 'autofocus' => true,
                 'autocomplete' => 'off',
+                'minlength' => 6,
+                'maxlength' => 6
```

Comment:
I vaguely remember that we removed browser-side validation some time in the past, because these were translated in the browser's locale, rather than the application's locale. Could you verify with the other forms, or with the client if need be?